CRM/Contribute/Form/Contribution.php

```diff
@@ -1683,7 +1683,7 @@ protected function submit($submittedValues, $action, $pledgePaymentID) {
       $action,
       $pledgePaymentID,
       $contribution->id,
-      (CRM_Utils_Array::value('option_type', $formValues) == 2) ? TRUE : FALSE,
+      ($formValues['option_type'] ?? 0) == 2,
```

Comment:
This is the only one I'm not 100% sure about @eileenmcnaughton thoughts?

Reply:
It's the same logic in both expressions.

Reply:
Yes - it first resolved $formValues['option_type'] to it's value OR 0 and then compares with 2

Reply:
Guess it just seemed odd to me resolving to 0 when CRM_Utils_Array::value would have resolved to NULL but i guess for this purpose they are interchangeable values

Reply:
Yep - probably NULL would have been more readable